changelogs/fragments/311-add-defaults-fields-snmp-community.yml

```
@@ -0,0 +1,2 @@
+minor_changes
```

This will fix the failing tests:
```suggestion
minor_changes:
```